.github/workflows/ci-it.yaml

```diff
@@ -32,9 +32,15 @@ jobs:
     strategy:
       fail-fast: true
     steps:
-      - uses: actions/checkout@v1
-        with:
-          submodules: true
+      - uses: actions/checkout@v2
+      - name: Checkout submodules
+        shell: bash
+        run: |
+          # If your submodules are configured to use SSH instead of HTTPS please uncomment the following line
+          # git config --global url."https://github.com/".insteadOf "git@github.com:"
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
```

Comment:
Why we need auth_header? As we are linking to open source only.

Reply:
This is a screenshot image when the previous run failed. 
<img width="1394" alt="WX20200227-074637@2x" src="https://user-images.githubusercontent.com/2892433/75398715-6b815c00-5935-11ea-9daf-58f91177367a.png">
This should be set before. This parameter will be taken after running the command. There is no in-depth study. This is the way to refer to the official action documentation.[https://github.com/actions/checkout#Checkout-submodules](url)

Reply:
My point is, you should be safe to run `git submodule update --init --force --recursive --depth=1` directly. I have run this command at local, we don't need that `auth_header`.

Reply:
@wu-sheng  Actions execute should be a bit different from local. For this successful example , the `auth_header` parameter be set in the before checkout step `actions/checkout@v2`, So all of the following need to take this parameter.

<img width="1072" alt="WX20200227-115704@2x" src="https://user-images.githubusercontent.com/2892433/75411012-c37d8a00-5958-11ea-8193-6cb33d654b02.png">

Reply:
Why should be different? I am not following. We are using the same parameter to download from GitHub too.

Could you explain your meaning of SHOULD?

Reply:
Don't copy the thing if that is not necessary.

Reply:
> Don't copy the thing if that is not necessary.

get it. 


> Why should be different? I am not following. We are using the same parameter to download from GitHub too.
> 
> Could you explain your meaning of SHOULD?

It is indeed that I understand it wrong, `auth_header` can be deleted for sw project, because english is not good, the description is not accurate, it is possible.